src/components/WebexMeeting/WebexMeeting.scss

```diff
@@ -11,4 +10,17 @@
   min-width: $WEBEX_MEETING_MIN_WIDTH;
 
   background: var(--wxc-meeting-background);
+
+  .member-roster-container {
+    align-self: start;
+    width: 20rem;
+    padding: 0.625rem;
+  }
+
+  .control-bar-container {
+    position: absolute;
```

Comment:
i don't think position absolute is required, the flex should take care of laying out this element correctly

Reply:
the flex would put the control bar side by side with the meeting and i can't add flex-direction: column because the roster has to be side by side with the meeting

Reply:
ok, then we have to add a parent for the video component (inmeeting, interstitialmeeting) and roster, with horizontal flex